src/main/java/com/cburch/logisim/file/XmlWriter.java

```diff
@@ -229,8 +229,8 @@ void addAttributeSetContent(Element elt, AttributeSet attrs, AttributeDefaultPro
             || (attr.equals(ProbeAttributes.PROBEAPPEARANCE) && !userModifiedOnly && val.equals(ProbeAttributes.APPEAR_EVOLUTION_NEW))) {
           final var a = doc.createElement("a");
           a.setAttribute("name", attr.getName());
-          if ("filePath".equals(attr.getName()) && outFilepath != null) {
-            final var outFP = Paths.get(outFilepath);
+          if ("filePath".equals(attr.getName()) && outFilePath != null) {
```

Comment:
I'd swap the order as null check is cheaper than string comparison. But that's minor thingh